job-runner.yml

```diff
@@ -4,6 +4,29 @@
   connection:     local
   gather_facts:   false
   become:         false
+  vars:
+    job_project_name: >-
+      {{
+          job_vars['__meta__']['deployer']['type'] ~ '-'
+        ~ job_vars['__meta__']['deployer']['scm_ref']
+      }}
+    job_template_name: >-
+      {{
+          job_vars['__meta__']['tower']['action'] ~ '-'
+        ~ job_vars['__meta__']['deployer']['type'] ~ '-'
+        ~ job_vars['__meta__']['deployer']['scm_ref'] ~ '-'
+        ~ job_vars['guid']
+      }}
+    job_forks:                 "{{ job_vars['__meta__']['tower']['forks'] | default(10) }}"
+    job_playbook:              "{{  job_vars['__meta__']['deployer']['entry_point'] }}"
+    job_virtualenv:   "{{  job_vars['__meta__']['deployer']['virtualenv'] | default('') }}"
+    _job_virtualenv:  "{{ '/opt/rh/virtualenvs/' ~ job_virtualenv if job_virtualenv else '' }}"
```

Comment:
Maybe allow both absolute paths and relative paths to the default '/opt/rh/virtualenvs/':

```yaml
    _job_virtualenv:  >-
      {{ job_virtualenv if job_virtualenv is match('^/')
      else ('/opt/rh/virtualenvs/' ~ job_virtualenv if job_virtualenv else '') }}

```

Reply:
Adjusted the PR to allow this.

Reply:
Quick comment `... is match('/')` automatically matches beginning of string, no need to anchor the regex with `^`